### PR TITLE
Add options for window

### LINKED ISSUE #4: Run on horizontal terminal
Will be good to have optino run horizontal split (float also).

### PR DESCRIPTION
**Pull request by EpsilonKu**
Should resolve #4 (too lazy to make float).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,14 @@ A customizable Neovim plugin to run code inside the editor
       dependencies = {
         'nvim-telescope/telescope.nvim',
         dependencies = { 'nvim-lua/plenary.nvim' }
-      }
+      }, 
+      config = function()
+        require("runner").setup({
```
Comment (MarcHamamji) [inline README.md]: Use single quotes for consistency

File: README.md [hunk 2]
```diff
@@ -39,7 +39,14 @@ A customizable Neovim plugin to run code inside the editor
...
+          position = "right", -- position of task window can be: top, left, right, bottom
```
Comment (MarcHamamji) [inline README.md]: Same, use single quotes

File: README.md [hunk 3]
```diff
@@ -55,7 +62,15 @@ A customizable Neovim plugin to run code inside the editor
...
+  Runner comes with the following defaults:
+  ```lua
+  {
+    position = "right", -- position of task window can be: top, left, right, bottom
+    width = 80,         -- width of window when position is left or right
+    height = 10,        -- height of window when position is top or bottom
+  }
+  ```
 
   Default handlers can be found [here](./lua/runner/handlers/init.lua).
```
Comment (MarcHamamji) [inline README.md:77]: Move this line to the next paragraph, after the table

File: README.md [hunk 4]
```diff
@@ -55,7 +62,15 @@ A customizable Neovim plugin to run code inside the editor
 
 ## Configuration
 
-  This plugin doesn't have a `setup()` method, but you can set your filetype handlers using the `set_handler()` method.
+  ### S`setup(options)`
+  Runner comes with the following defaults:
+  ```lua
+  {
```
Comment (MarcHamamji) [inline README.md]: I would prefer you run the setup function in this block to make it more explicit and to have a copy-pastable example.

File: README.md [hunk 5]
```diff
@@ -55,7 +62,15 @@ A customizable Neovim plugin to run code inside the editor
...
+    position = "right", -- position of task window can be: top, left, right, bottom
```
Comment (MarcHamamji) [inline README.md]: Single quotes

File: README.md [hunk 6]
```diff
@@ -55,7 +62,15 @@ A customizable Neovim plugin to run code inside the editor
 
 ## Configuration
 
-  This plugin doesn't have a `setup()` method, but you can set your filetype handlers using the `set_handler()` method.
+  ### S`setup(options)`
```
Comment (MarcHamamji) [inline README.md]: Typo, remove the `S`

File: lua/runner/config.lua [hunk 7]
```diff
@@ -0,0 +1,17 @@
+local M = {}
+
+local defaults = {
+  position = "right", -- options: top, left, right, bottom
```
Comment (MarcHamamji) [inline lua/runner/config.lua]: Single quotes

File: lua/runner/config.lua [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+  width = 80,         -- width of window when position is left or right
+  height = 10,        -- height of window when position is top or bottom
+}
+
+M.options = {}
+
+M.setup = function(options)
+  M.options = vim.tbl_deep_extend("force", {}, defaults, options or {})
```
Comment (MarcHamamji) [inline lua/runner/config.lua]: Single quotes

**Review by MarcHamamji — CHANGES_REQUESTED** [review]
Thank you very much. I now realize that I need to setup StyLua as soon as possible 😅.
Also, I think that a floating window wouldn't be very helpful, I don't see how it would be used practically.

**Comment by EpsilonKu** [comment]
@MarcHamamji Ok, can u check it now? Sorry for a lot of comments. My WSL render symbols weirdly. Basically I see result normally only on Github.

File: README.md [hunk 9]
```diff
@@ -54,12 +60,19 @@ A customizable Neovim plugin to run code inside the editor
   ```
 
 ## Configuration
+  #### `setup(options)`
+  Runner comes with the following defaults:
+  ```lua
```
Comment (MarcHamamji) [inline README.md:67]: I know I'm a bit picky but can you please remove the indentation inside this block (only the code)? When you render the markdown, this block is indented whereas the others are not. Thanks.

**Review by MarcHamamji — CHANGES_REQUESTED** [review]

**Comment by MarcHamamji** [comment]
> @MarcHamamji Ok, can u check it now? Sorry for a lot of comments. My WSL render symbols weirdly. Basically I see result normally only on Github.

No worries :)

**Comment by EpsilonKu** [comment]
Ok, I also added spaces between titles.

**Review by MarcHamamji — APPROVED** [review]

**Pull request merged**